### PR TITLE
Documentation reorganization

### PR DESCRIPTION
**Pull request by iranzo**
# Description

This PR:
- reorganizes the documentation into sections
- Adds inks to code blocks

Preview at: https://deploy-preview-321--baremetal-deploy.netlify.app/ 

Fixes # (issue)

## Type of change

Please select the appropiate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [x] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [x] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: documentation/modules/ipi-install-creating-a-disconnected-registry.adoc [hunk 1]
```diff
@@ -108,65 +124,105 @@ $ sudo podman create \
...
+On the provisioner node, the `install-config.yaml` file should use the newly created pull-secret from the `pull-secret-update.json` file. The `install-config.yaml` file must also contain the disconnected registry node's certificate and registry information.
 
 .Procedure
 
 . Add the disconnected registry node's certificate to the `install-config.yaml` file. The certificate should follow the `"additionalTrustBundle: |"` line and be properly indented, usually by two spaces.
 +
+[source,bash]
 ----
-echo "additionalTrustBundle: |" >> install-config.yaml
-sed -e 's/^/  /' /opt/registry/certs/domain.crt >> install-config.yaml
```
Comment (rlopez133) [inline documentation/modules/ipi-install-creating-a-disconnected-registry.adoc:159]: This procedure mentions to add the disconnected registry node's ceriticate to the install-config.yaml. However, it makes the assumption that the domain.crt already resides in the provisioner host.

Prior to 
```
[kni@provisioner ~]$ echo "additionalTrustBundle: |" >> install-config.yaml
```
We need a step like the following:
```
[kni@provisioner ~]$ scp user@registry.example.com:/opt/registry/certs/domain.crt /home/kni
```
@iranzo ^ keep me honest, but I believe that is accurate syntax. 

So step 1 should be:
```
[kni@provisioner ~]$ scp user@registry.example.com:/opt/registry/certs/domain.crt /home/kni
[kni@provisioner ~]$ echo "additionalTrustBundle: |" >> install-config.yaml
[kni@provisioner ~]$ sed -e 's/^/  /' /home/kni/domain.crt >> install-config.yaml
```

File: documentation/modules/ipi-install-config-yaml.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  replicas: 3
+  platform:
+    baremetal: {}
+platform:
+  baremetal:
+    apiVIP: <api-ip>
+    ingressVIP: <wildcard-ip>
+    dnsVIP: <dns-ip>
+    provisioningNetworkInterface: <NIC1>
+    provisioningNetworkCIDR: <CIDR> # i.e. 172.22.0.1/24
```
[the remark this answers is not in the document]
Comment (iranzo) [reply documentation/modules/ipi-install-config-yaml.adoc:26]: Will be in next push

File: documentation/modules/ipi-install-creating-a-disconnected-registry.adoc (hunk 1 above)
Comment (iranzo) [reply documentation/modules/ipi-install-creating-a-disconnected-registry.adoc:159]: going in next push

**Pull request merged**